wpilibc/src/main/native/include/frc/PneumaticsBase.h

```diff
@@ -22,59 +22,214 @@ class PneumaticsBase {
 
+  /**
+   * If supported by the device, enables the compressor in analog mode. This
+   * mode uses an analog pressure sensor to cycle the compressor. The compressor
+   * will turn on when the pressure drops below {@code minPressure} and will
+   * turn off when the pressure reaches {@code maxPressure}. This mode is only
+   * supported by the REV PH with the REV Analog Pressure Sensor.
+   *
+   * On CTRE PCM, this will enable digital control.
+   *
+   * @param minPressure The minimum pressure. The compressor will turn on
+   * when the pressure drops below this value.
+   * @param maxPressure The maximum pressure. The compressor will turn
+   * off when the pressure reaches this value.
+   */
   virtual void EnableCompressorAnalog(
       units::pounds_per_square_inch_t minPressure,
       units::pounds_per_square_inch_t maxPressure) = 0;
 
+  /**
+   * If supported by the device, enables the compressor in hybrid mode. This
+   * mode uses both a digital pressure switch and an analog pressure sensor to
+   * cycle the compressor. This mode is only supported by the REV PH with the
+   * REV Analog Pressure Sensor.
+   *
+   * The compressor will turn on when \a both:
+   *
+   * - The digital pressure switch indicates the system is not full AND
+   * - The analog pressure sensor indicates that the pressure in the system
+   * is below the specified minimum pressure.
+   *
+   * The compressor will turn off when \a either:
+   *
+   * - The digital pressure switch is disconnected or indicates that the system
+   * is full OR
+   * - The pressure detected by the analog sensor is greater than the specified
+   * maximum pressure.
+   *
+   * On CTRE PCM, this will enable digital control.
+   *
+   * @param minPressure The minimum pressure. The compressor will turn on
+   * when the pressure drops below this value and the pressure switch indicates
+   * that the system is not full.
+   * @param maxPressure The maximum pressure. The compressor will turn
+   * off when the pressure reaches this value or the pressure switch is
+   * disconnected or indicates that the system is full.
+   */
   virtual void EnableCompressorHybrid(
       units::pounds_per_square_inch_t minPressure,
       units::pounds_per_square_inch_t maxPressure) = 0;
 
+  /**
+   * Returns the active compressor configuration.
+   *
+   * @return The active compressor configuration.
+   */
   virtual CompressorConfigType GetCompressorConfigType() const = 0;
 
+  /**
+   * Sets solenoids on a pneumatics module.
+   *
+   * @param mask mask
+   * @param values values
+   */
   virtual void SetSolenoids(int mask, int values) = 0;
 
+  /**
+   * Gets solenoid values.
+   *
+   * @return values
+   */
   virtual int GetSolenoids() const = 0;
 
+  /**
+   * Get module number for this module.
+   *
+   * @return module number
+   */
   virtual int GetModuleNumber() const = 0;
 
+  /**
+   * Get the disabled solenoids.
+   *
+   * @return disabled list
```

Comment:
The return type doesn't seem to be a list. Assuming it's a bitmask, document as such.